test/integration/integration_test.go

```diff
@@ -156,7 +112,7 @@ func getLogLevel() (level int) {
 // setup sets up integration tests
 func (i *integrationTest) setup() {
 	var err error
-	i.engineClient, err = dockerClient(dockerConfig())
+	i.engineClient, err = dockerapi.NewEnvClient()
```

Comment:
This is fixed upstream at https://github.com/docker/engine-api/commit/cf82c64276ebc2501e72b241f9fdc1e21e421743 .  Let me look at bumping the vendored packages.